Support multiple paths with asterisks (#1860)
kyma-project/api-gateway: docs/user/custom-resources/apirule/v2alpha1/04-10-apirule-custom-resource.md

| **status.state**       | Defines the reconciliation state of the APIRule. The possible states are `Ready`, `Warning`, `Error`, `Processing`, or `Deleting`. |
| **status.description** | Contains a detailed description of **status.state**.                                                                                         |

> **Significance of Rules Path Order:**  

Reviewer: We should add this to release notes as this is a significant feature

Reviewer 2: RN added

> **Significance of Rules Path Order:**  

Reviewer 3: ```suggestion
### Significance of Rules Path Order
```

> Operators `{*}` and `{**}` allow to define a single APIRule that matches multiple request paths.
> However, this also introduces the possibility of path conflicts.
> A path conflict occurs when two or more APIRule resources match the same path. This is why rule order is important.

Reviewer 3: ```suggestion
Operators `{*}` and `{**}` allow you to define a single APIRule that matches multiple request paths.
However, this also introduces the possibility of path conflicts.
A path conflict occurs when two or more APIRule resources match the same path. This is why the order of rules is important.
```

> Rules defined earlier in the list have a higher priority than those defined later. Therefore, we recommend defining `rules` from the most specific `path` to the most general.

Reviewer 3: ```suggestion
Rules defined earlier in the list have a higher priority than those defined later. Therefore, we recommend defining rules from the most specific path to the most general.
```

> Example of a valid `rules.path` order, listed from the most specific to the most general:

Reviewer 3: ```suggestion
See an example of a valid **rules.path** order, listed from the most specific to the most general:
```
